evennia/utils/test_resources.py

```diff
@@ -157,3 +157,11 @@ def tearDown(self):
         self.account.delete()
         self.account2.delete()
         super().tearDown()
+
+class LocalEvenniaTest(EvenniaTest):
+    account_typeclass = settings.BASE_ACCOUNT_TYPECLASS
```

Comment:
This needs a documentation string to explain the purpose of this as a testing base.